### PR TITLE
Update thanks.md

### PR DESCRIPTION
**Pull request by IvenBach**

### CODE REVIEW

File: thanks.md [hunk 1]
```diff
@@ -2,5 +2,17 @@
 
 In this file, you can read about where and how people use Rubberduck:
 
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
```
Comment (retailcoder) [inline thanks.md]: I think we could use a `<!-- add new content above -->` comment here (we want the most-recent on top, right?)

File: thanks.md [hunk 2]
```diff
@@ -2,5 +2,17 @@
...
+
+When I first started I didn't think I'd be able to successfully contribute, and honestly still don't. But if I didn't start when I did I wouldn't be where I'm at now. I'm glad my past self (exactly 1 year ago yesterday) was willing to step outside my comfort zone and ask about contributing to OSS. Just remember that you #GottaStartsSomewhere.
+
+Thanks to all those involved with RD enough for helping me learn a better way to code.
+
+[@IvenBach](https://github.com/IvenBach)
```
Comment (retailcoder) [inline thanks.md]: Should there be a date next to the name?

File: thanks.md (hunk 1 above)
Comment (Vogel612) [reply thanks.md]: seems like a good suggestion. We might even add a template there :)

File: thanks.md (hunk 2 above)
Comment (Vogel612) [reply thanks.md]: Is theoretically visible in the blame, but might as well add it

File: thanks.md [hunk 3]
```diff
@@ -2,5 +2,22 @@
...
+- Understand the differences between `ByVal` & `ByRef` through [Code Inspections](https://github.com/rubberduck-vba/Rubberduck/wiki/Code-Inspections).
+- Learn about and successfully writing [Unit Tests](https://github.com/rubberduck-vba/Rubberduck/wiki/Unit-Testing) that allow me to make changes without fear of unintentionally breaking functionality.
+- Understand the differences between Early and Late binding.
+- Learn and use [Dependency Injection]( https://en.wikipedia.org/wiki/Dependency_injection).
+- Programming to an abstraction and not a concretion.
+- How to properly Refactor code.
+- Write Self documenting code (comments should explain the why not what).
+- Achieve consistent indentation.
+- Referencing external assemblies.
+- And a gaggle of other stuff too...
```
Comment (retailcoder) [inline thanks.md]: IMO the list should be removed, it's already simply/vaguely summarized in the preceding sentence.

I'm expecting the average entry to read something like this (that one's an *actual* SO comment, somewhere):

> Thanks a bunch! A simple, straightforward unit testing tool for VBA is a godsend!

Or whatever fits a SO comment... or two :wink:

**Pull request merged**